### ISSUE TITLE
Better UI for calendar invites

### ISSUE THREAD

**Issue by mu22le** [protz/thunderbird-conversations#2074]
I am not sure if suggestion is for thunderbird-conversations or should rather go to Thunderbird directly. Apologies in advance if that is the case.

* **Thunderbird version: 115.6.0 (64 bit)**
* **Thunderbird Conversation version: 4.1.5**
* **Operating System: Linux Mint 21.2 Cinnamon**

**What did you do?**
When I receive a message containing a calendar invite, I'll click on it in the message list.

**What did you expect to happen?**
I would expect a summary of the invite details to be shown in the light blue bar at the top, together with the accept/decline buttons. It would also be nice if I could click on the date and be taken directly to the calendar tab, in order to see the event in context.

**What actually happened?**
The invite details are only shown at the bottom of the message, and they are folded, so that I need to click on them to expand the box. When I do that, the box invariably goes over the bottom of the message (the size of the message panel was calculated to only fit the message with the folded info box and is not recomputed) so I have to scroll further to see the invitation info. Then I have to scroll all the way back to the top of the message to accept/decline the invite.
![Screenshot from 2024-01-22 15-21-17](https://github.com/thunderbird-conversations/thunderbird-conversations/assets/3257464/e062dc89-acdf-4713-808b-51f396f293b3)

**Comment by funplayertv** [comment]
it seems to me that this is a little related to my request: https://github.com/thunderbird-conversations/thunderbird-conversations/issues/2010

**Comment by mu22le** [comment]
Yes, #2010 is related, but my proposal (add time and date in the top bar and make it a link to the event in the calendar) is a bit more wide-ranging. I can close this report and add my request at the bottom of the other one, if you prefer.

**Comment by MLopez-Ibanez** [comment]
The UI needs a revamp. It should also show which events conflict with the event proposed so one can decide whether to accept or not the invitation.

**Comment by Standard8** [comment]
The current UI is based on Thunderbird's current UI. I know they have some work in the future to improve that UI, but I don't know if when/if it will ship, or if we'd be able to include it here easily or not (I suspect not).
